src/server/game/Loot/LootMgr.cpp

```diff
@@ -301,53 +308,53 @@ bool LootStoreItem::Roll(bool rate, Player const *player, Loot& loot, LootStore
 // Checks correctness of values
 bool LootStoreItem::IsValid(LootStore const& store, uint32 entry) const
 {
-    if (group >= 1 << 7)                                     // it stored in 7 bit field
+    if (groupid >= 1 << 7)                                     // it stored in 7 bit field
     {
-        sLog->outErrorDb("Table '%s' entry %d item %d: group (%u) must be less %u - skipped", store.GetName(), entry, itemid, group, 1 << 7);
+        sLog->outErrorDb("Table '%s' Entry %d Item %d: GroupId (%u) must be less %u - skipped", store.GetName(), entry, itemid, groupid, 1 << 7);
         return false;
     }
 
-    if (mincountOrRef == 0)
+    if (reference == 0)
     {
         sLog->outErrorDb("Table '%s' entry %d item %d: wrong mincountOrRef (%d) - skipped", store.GetName(), entry, itemid, mincountOrRef);
         return false;
     }
 
-    if (mincountOrRef > 0)                                  // item (quest or non-quest) entry, maybe grouped
+    if (reference > 0)                                  // item (quest or non-quest) entry, maybe grouped
```

Comment:
```suggestion
    if (reference == 0)                                  // item (quest or non-quest) entry, maybe grouped
```

Reply:
no here ref must be > 0 ref == 0 is early

Reply:
Changed according to this line https://github.com/TrinityCore/TrinityCore/commit/55b43c67519359f0e5a96004c393898b3c62add3#diff-469201924aa4feec87dd70a2bf60bb14R310